### PR TITLE
HLM-494

### PR DESCRIPTION
**Pull request by saiprakash-v**
@prasanna-egov @egov-joy 
This PR is raised to get all schemas available per tenant
**Assumption:** Tenant will have access to all the schemas, certificates
This api will be fetching only the schemas generated by the user who generated them (based on osOwner attribute set during creation of schemas). So, I had to create schemas with admin-api client creds, for it to be fetched later.

### CODE REVIEW

**Comment by prasanna-egov** [comment]
Swagger is not updated. Once it is done we can merge.

File: backend/vc-management-service/src/controllers/schema.controller.js [hunk 1]
```diff
@@ -19,6 +19,23 @@ async function createSchema(req, res) {
...
 }
 
+async function getSchema(req, res) {
+    try {
+        const token = req.header("Authorization");
+        const schemaId = req.params.schemaId;
+        const fetchSchemaResponse = await sunbirdRegistryService.getSchema(token, schemaId);
+        res.status(200).json({
+            message: "Successfully fetched Schema",
+            getSchemaResponse: fetchSchemaResponse
```
Comment (prasanna-egov) [inline backend/vc-management-service/src/controllers/schema.controller.js:29]: Change the payload to send out only the schema response from sunbird, or even a subset of the response

Comment (saiprakash-v) [reply backend/vc-management-service/src/controllers/schema.controller.js:29]: Sure. I will change response and update swagger doc as well

Comment (prasanna-egov) [reply backend/vc-management-service/src/controllers/schema.controller.js:29]: This is still not done. @saiprakash-v

Comment (saiprakash-v) [reply backend/vc-management-service/src/controllers/schema.controller.js]: Sorry. I've committed changes but didn't push them earlier. Please review now

**Pull request merged**